projects/sbb-esta/angular-business/header/src/header/header.component.html

```diff
@@ -21,21 +21,27 @@
   </div>
   <template #navigationButtons></template>
 </div>
-<div class="sbb-header-usermenu">
-  <ng-content select="sbb-usermenu"></ng-content>
-</div>
 
-<!-- If an icon is provided, use that, otherwise use the sbb logo -->
-<!-- TODO: make cleaner -->
-<div #iconContent class="sbb-header-logo">
-  <ng-content select="[brand],.brand"></ng-content>
-</div>
-<div *ngIf="!iconContent.childElementCount" class="sbb-header-logo">
-  <svg viewBox="0 0 80 21" xmlns="http://www.w3.org/2000/svg">
-    <path d="M0 0h59.233v20.603H0V0z" fill="#EC0000"></path>
-    <path
-      d="M35.186 17.02h3.75l-5.047-5.163h6.265v5.163h2.96v-5.163h6.267l-5.05 5.163h3.752l6.427-6.708-6.426-6.73h-3.752l5.05 5.185h-6.266V3.583h-2.96v5.184h-6.267l5.047-5.184h-3.75l-6.43 6.73 6.43 6.707"
-      fill="#FFF"
-    ></path>
-  </svg>
+<div class="sbb-header-container-end">
+  <div class="sbb-header-additional-content">
+    <ng-content select=".additional-header-content"></ng-content>
+  </div>
+  <div class="sbb-header-usermenu">
+    <ng-content select="sbb-usermenu,.sbb-usermenu"></ng-content>
+  </div>
+
+  <!-- If an icon is provided, use that, otherwise use the sbb logo -->
+  <!-- TODO: make cleaner -->
+  <div #iconContent class="sbb-header-logo" [class.hidden-logo]="!iconContent.childElementCount">
```

Comment:
Use [hidden] instead of [class.hidden-logo] and remove the css code

Reply:
good point, fixed this with the lastest commit